libs/typography/src/lib/typography.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { ColorVariants, SizeVariants } from '@learnbase-ui/global-types';
+import { typographyStyles } from './typography.class';
+import { twMerge } from 'tailwind-merge';
+import '../styles.scss';
+
+export type TypographySizeProps = Exclude<SizeVariants, '2xl' | '3xl'>;
```

Comment:
If we always remove 2xl and 3xl can we add another size type something like `componentSize`?